coordinator/src/models/fhir/medication-request.ts

```diff
@@ -11,17 +10,17 @@ export enum CourseOfTherapyTypeCode {
 }
 
 export const CourseOfTherapyType = Object.freeze({
```

Comment:
seems like `Object.freeze` doesnt stop me from changing values within the codeable concept. for this to be useful wouldnt we like it to stop any changes at all?

Reply:
It doesn't but neither would
```
const SOME_CONSTANT = {
  key1: "value1",
  key2: "value2"
}
```
so it's as "immutable" as any object constant

Reply:
Replaced with equivalent top level consts